### DISCUSSION TITLE
'End' ⇄ 'Start' of segment at its creation stage

### DISCUSSION THREAD

**Discussion by evd316** [kitadai31/revanced-patches-android6-7#83] [Q&A]
Hi, I already wrote here: github.com/orgs/ReVanced/discussions/1061.
I just want to ask you too, so that this definitely gets into your patches if they make a change and update.
![зображення_viber_2024-08-14_02-40-37-859](https://github.com/user-attachments/assets/a6d250e7-37b4-4476-9bcd-64803d27b54c)

**Comment by kitadai31** [comment]
solved in the ReVanced repo

**Comment by evd316** [comment]
They suggested reducing the font size in the system. I have it 'default' (like most users). I reduced the font, yes - it helped, but it became impossible to use the phone. Their advice is trashed. Need to change the layout and reduce the number of letters in words or leave it without translation. I can find synonyms with fewer characters in words.
How many characters per line are needed in this case and can you do it?

**Reply by kitadai31** [reply]
Sorry, I don't know how the button layout is determined

**Reply by evd316** [reply]
But can you actually change the translation of the words?
There you can insert "Старт, Стоп and Назад", the number of characters is much smaller.
